### PR TITLE
Bug 1309901 - Use spot instances for ad-hoc clusters

### PR DESCRIPTION
**Pull request by fbertsch**
@jezdez r?

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-analysis-service/41)

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: atmo/settings.py [hunk 1]
```diff
@@ -91,6 +91,7 @@
     # setup bootstrap action depends on it to autotune the cluster.
     'MASTER_INSTANCE_TYPE': 'c3.4xlarge',
     'SLAVE_INSTANCE_TYPE': 'c3.4xlarge',
```
Comment (jezdez) [inline atmo/settings.py]: Let's rename that setting to `WORKER_INSTANCE_TYPE` to math its use below

File: atmo/settings.py [hunk 2]
```diff
@@ -91,6 +91,7 @@
...
+    'CORE_SPOT_BID': '0.84',
```
Comment (jezdez) [inline atmo/settings.py]: If yes, wrapping this into a `config` call that allows us to set this via an environment variable (see other settings for an example) would be useful

**Comment by fbertsch** [comment]
@jezdez made the changes you recommended, RFAL.

**Comment by vitillo** [comment]
@fbertsch How has this been tested? How much longer does it take on average for a cluster to bootstrap now?

**Comment by jezdez** [comment]
I have only done code review, not functional testing but you can try it on https://atmo-staging.herokuapp.com which is automatically deployed with master

**Comment by jezdez** [comment]
FTR We missed on case of the old variable name, fixed in https://github.com/mozilla/telemetry-analysis-service/commit/732b2f75d1493581f8331c7f8a263ba6aa113cc5

**Pull request merged**